repo/packages/B/bookkeeper/0/marathon.json.mustache

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "{{bookkeeper.app_id}}",
+  "cpus":{{bookkeeper.cpus}},
+  "mem": {{bookkeeper.mem}},
+  "instances": {{bookkeeper.instances}},
+  "constraints": [["hostname", "UNIQUE"]],
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/bk/journal",
+        "hostPath": "/mnt/journal",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/index",
+        "hostPath": "/mnt/index",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/bk/ledgers",
+        "hostPath": "/mnt/ledgers",
+        "mode": "RW"
+      },
+  
+      {
+        "containerPath": "persist",
+        "mode": "RW",
+        "persistent": {
+          "size": {{bookkeeper.psize}}
+        }
+      }
+    ],
+
+    "docker": {
+      "image": "{{resource.assets.container.docker.bookkeeper}}",
+      "network": "HOST"
+    },
+    
+    "env": {
+        "ZK": {{bookkeeper.zk_client}},
+        "BPORT": {{bookkeeper.service_port}}
```

Comment:
Got it. Yes, bookie is on the host network, it prefered a port - 3181, but this is not a fixed port. 
Will change it to include a portdefinition field. Thanks.